ios/examples/run-parallel-tests/parallel_test.py

```diff
@@ -0,0 +1,65 @@
+from appium import webdriver
+from appium.webdriver.common.mobileby import MobileBy
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+import time
+import os, json, threading
+
+
+config_file_path = os.path.join(os.path.dirname(__file__), "config.json")
+print("Config file path= %s" % (config_file_path))
```

Comment:
```suggestion
print("Config file path: %s" % (config_file_path))
```

Reply:
Or maybe some more proper messaging like `print(Path to the config file: %s % (config_file_path))`